babel.config.js

```diff
@@ -1,7 +1,7 @@
 const BABEL_ENV = process.env.BABEL_ENV;
 const IS_TEST = BABEL_ENV === "test"
 const ignore = IS_TEST ? [] : ['test/**/*.js']
-const targets = IS_TEST ? { node: "current" } : { browsers: "defaults" }
+const targets = IS_TEST ? { node: "current" } : {}
```

Comment:
I'm guessing it works, but I don't understand why.  Is there an article or something that you found that helped you fix it? If so, maybe mentioning it in the PR description?